routes/preview.js

```diff
@@ -80,16 +82,63 @@ router.get('/prepublish/behandelingen/:zittingIdentifier', async function(req, r
 */
 router.get('/prepublish/notulen/:zittingIdentifier', async function(req, res, next) {
   try {
-    const zitting = await getZittingForNotulen( req.params.zittingIdentifier );
-    const {html, errors} = await extractNotulenContentFromZitting(zitting);
+    const {html, errors} = await constructHtmlForMeetingNotes(req.params.zittingIdentifier);
     return res.send( { data: { attributes: { content: html, errors }, type: "imported-notulen-contents" } } ).end();
-  } catch (err) {
-    console.log(err);
-    const error = new Error(`An error occurred while fetching contents for prepublished notulen ${req.params.zittingIdentifier}: ${err}`);
+  }
+  catch(e) {
+    console.error(e);
+    const error = new Error(`An error occurred while fetching contents for prepublished notulen ${req.params.zittingIdentifier}: ${e}`);
     // @ts-ignore
     error.status = 500;
     return next(error);
   }
 });
 
+router.post('/extract-previews', async function (req, res, next) {
```

Comment:
this is a new json:api compliant route that can be used in the future. should work like this in ember:
```
const extractPreview = this.store.createRecord('extract-preview');
extractPreview.treatment = treatment;
yield extractPreview.save();
// extractPreview.html is now set

```

Reply:
```
import Model from '@ember-data/model';
import { attr, belongsTo, hasMany } from '@ember-data/model';
export default class ExtractPreviewModel extends Model {
  @belongsTo('behandeling-van-agendapunt') treatment;
  @attr html;
}
```
tested with this model